version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.6.4-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.6.5-SNAPSHOT"
```

Comment:
Do not touch this file (see https://github.com/VirtusLab/unicorn/blob/v0.6.x-slick-2.1.x/CONTRIBUTING.md#releasing).